README.md

```diff
@@ -233,7 +233,7 @@ Copy $CELEBORN_HOME/spark/*.jar to $SPARK_HOME/jars/
 #### Spark Configuration
 To use Celeborn,the following spark configurations should be added.
 ```properties
-spark.shuffle.manager org.apache.spark.shuffle.celeborn.RssShuffleManager
+spark.shuffle.manager org.apache.spark.shuffle.celeborn.SparkShuffleManager
```

Comment:
Although we have migration doc, add comment for this configuration change in ReadMe file is good/still need to users.

Reply:
Done